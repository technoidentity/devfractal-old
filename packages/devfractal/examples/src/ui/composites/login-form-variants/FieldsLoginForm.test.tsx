import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { FieldsLoginForm } from './FieldsLoginForm'

test.skip('Fields login form', () => {
  const component: ReactTestRenderer = renderer.create(<FieldsLoginForm />)
  expect(component.toJSON()).toMatchSnapshot()
})
