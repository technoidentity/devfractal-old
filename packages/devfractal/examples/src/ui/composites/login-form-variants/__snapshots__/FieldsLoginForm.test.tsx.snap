// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fields login form 1`] = `
<div
  className="container"
>
  <form
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label
        className="label"
      >
        Username
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          name="username"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="field"
    >
      <label
        className="label"
      >
        Password
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
      </div>
    </div>
    <div
      className="field is-grouped is-grouped-centered"
    >
      <div
        className="control"
      >
        <button
          className="button is-info"
          type="submit"
        >
          Submit
        </button>
      </div>
      <div
        className="control"
      >
        <button
          className="button is-danger"
          type="reset"
        >
          Reset
        </button>
      </div>
    </div>
  </form>
</div>
`;
